web/packages/teleport/src/DesktopSession/useDesktopSession.tsx

```diff
@@ -77,12 +110,13 @@ export default function useDesktopSession() {
           .fetchDesktop(clusterId, desktopName)
           .then(desktop => setHostname(desktop.name)),
         userService.fetchUserContext().then(user => {
-          setClipboardSharingEnabled(
-            user.acl.clipboardSharingEnabled && isUsingChrome
-          );
+          setClipboardSharingState(prevState => ({
+            ...prevState,
+            canShare: user.acl.clipboardSharingEnabled && isUsingChrome,
```

Comment:
TBH `canShare` this shouldn't be stored in the state (since it is calculated from other state), you could just generate it dynamically when needed.